samples/README.MD

```diff
@@ -0,0 +1,46 @@
+# Samples, Snippets, and How-To Guides
+
+Developers like to learn by looking at code, and so the Azure SDK comes with a myriad of code samples in the form of short code snippets, sample applications, and how-to guides. This document describes where to find all these resources.
+
+## Structure of the Repository
+The Azure SDK repository is organized in the following folder structure, with the main sample locations highlighted using **bold** font.
+
+`/samples` - this folder<br>
+&nbsp;&nbsp;&nbsp;&nbsp;`README.md` ```// (this file)```<br>
+`/sdk` ```// folder containing sources for all SDK packages```<br>
+&nbsp;&nbsp;&nbsp;&nbsp;`/<service>` ```// e.g. storage```<br>
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`/<package>` ```// e.g. Azure.Storage.Blobs```<br>
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**`README.md` ```// package READMEs contain hello world samples```**<br>
```

Comment:
it's not C# you don't have to use `//` for folder descriptions

Reply:
It looks very busy right now 
![image](https://user-images.githubusercontent.com/1697911/68150358-69ae0b80-fef4-11e9-83ba-0ffca17ed4f3.png)

Reply:
What do you think I should do?

Reply:
`/samples` - this folder<br>
&nbsp;&nbsp;&nbsp;&nbsp;`README.md` - (this file)<br>
`/sdk` - folder containing sources for all SDK packages<br>
&nbsp;&nbsp;&nbsp;&nbsp;`/<service>` - e.g. storage<br>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`/<package>` - e.g. Azure.Storage.Blobs<br>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**`README.md`** - package READMEs contain hello world samples<br>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**`/samples`** - package-specific samples</br>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`/src`<br>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`/tests`<br>